pkg/config/config.go

```diff
@@ -76,6 +76,9 @@ const (
 
 	// environment variable for the Git rewrite setting
 	useGitRewriteRule = "GIT_ENABLE_REWRITE_RULE"
+
+	// environment variable to hold vulnerability count limit
+	VulnerabilityCountLimitEnvVar = "VULNERABILITY_COUNT_LIMIT"
```

Comment:
how this one works? e.g. what would be a desired limit and why we need this.

Also, it seems we have zero docs on this PR, we need to document this. I can help you, but pls looks at how we have the docs for the different CRs.

Reply:
I have kept the desired limit as 50 if this environment variable is not specified.
Yup we need to add `VULNERABILITY_COUNT_LIMIT ` to the docs.

Reply:
Correct, intention is to limit the result size (which is limitted by the Pods termination message, and the BuildRun status size.